### DISCUSSION TITLE
Enabling IPv6 access

### DISCUSSION THREAD

**Discussion by traylenator** [hassio-addons/addon-uptime-kuma#114] [Q&A]
Hi

Thanks for making this add-on.

I wanted to be able to ping a IPv6 address from update-kuma.

Generally IPv6 is working and enabled. In particular using the Advanced SSH plugin I can ping a IPv6 only address.

Attaching to the update kuma container them the same ping declares that as not possible.

This container only has IPv4.

I note that the SSH addon uses `host_network: true` in it's configuration. I've not tried this yet.

Any ideas how to enable IPv6 in the update-kuma container?

Steve.

**Comment by traylenator** [comment]
https://github.com/home-assistant/supervisor/issues/2133

Gives a summary of IPv6 status within containers. Think I've answered my own question. 

Unless running with host_network is possible?

**Comment by tonyisnewhuman** [comment]
I modify config.yaml configuration file, added host_network: true option, after reinstall the update-kuma plugin, successfully used the host network, supported IPv6, and was able to monitor IPv6 services, but I was unable to update. After the update, IPv6 support became invalid.

![1704868665654](https://github.com/hassio-addons/addon-uptime-kuma/assets/32791116/5c519777-6068-4a12-8fe9-206f4b23e628)
![1704868745617](https://github.com/hassio-addons/addon-uptime-kuma/assets/32791116/eb175cc3-3fb9-42be-882b-6ca752365907)
